repl/repl.go

```diff
@@ -228,6 +228,9 @@ func BeginRepl(args []string, version string) {
 			panic(err)
 		}
 		fmt.Printf("Hello %s, welcome to the ABS (%s) programming language!\n", user.Username, version)
+		if newver, update := util.UpdateAvailable(version); update {
```

Comment:
this will run every time someone launches the repl -- which can slow you down / be annoying. Can we do so that it runs every once in a while? We can even just randomize it (generate number n between 1 and 100) and only trigger it when  n is < 10 or similar. What do you think?

Reply:
Yeah, I can do a 10%-frequency check, but since there's also a `--check-update` command-line flag (which should always do the check), I'll do the frequency check in `repl.BeginRepl` instead.